### DISCUSSION TITLE
Quaility Histogram font size?

### DISCUSSION THREAD

**Discussion by DrBwts** [cnr-isti-vclab/meshlab#955] [Q&A]
HI -  I have been using MeshLab from some mesh anaylsis & will be publishing this data. I would like to ba able to change the font size of the Quaility Histogram. Is this possible? Its way too small & know that this will be pointed out straight away by any reviewer.

Nic

**Comment by alemuntoni** [comment]
Can you share a screenshot? Are you using a 4k hdpi monitor?

**Reply by DrBwts** [reply]
Not a 4k hdpi & the image will be used for a publication so the font needs to be clear & readable when printed

https://i.imgur.com/nXtaJlO.png

**Reply by alemuntoni** [reply]
Ok, I added a "text font size" parameter to the show quality decorator, that can be changed in the bottom-right area of the meshlab window.
You can try it by downloading the last nightly built by [github actions](https://github.com/cnr-isti-vclab/meshlab/actions) (select your platform and download the version you prefer).

**Comment by DrBwts** [comment]
Hi - thanks, I clicked on the link you sent but its not clear to me how to
download the build.

I clicked on
https://github.com/cnr-isti-vclab/meshlab/actions/runs/751350075 & then
onto the WIndows double precision option. This just seems to give me lists
of information with no link to download the actual program.

**Reply by alemuntoni** [reply]
You can find the list of artifact in the bottom of the page you linked.

![image](https://user-images.githubusercontent.com/6656168/115002767-666acd80-9ea5-11eb-86c0-351bffcb6797.png)

**Comment by DrBwts** [comment]
Ahhh yes thank you

**Comment by DrBwts** [comment]
Hi, I downloaded & run but get the following error (I'm on a Win 10 x64
machine)

[image: image.png]

**Reply by alemuntoni** [reply]
Cannot see the error. Please reply from github and not via email.

**Comment by DrBwts** [comment]
Hi that error,

![image](https://user-images.githubusercontent.com/22524334/115008937-a29d2e80-9ea3-11eb-8d8c-c3ebc9fc48a6.png)

**Reply by alemuntoni** [reply]
Could you please try to reinstall the vc_redist.exe that is inside the meshlab directory?

**Comment by DrBwts** [comment]
OK reinstalled  VC_redist.exe and its starting up but I cant find the dialogue to change the font size,

![image](https://user-images.githubusercontent.com/22524334/115032796-a0e26380-9ec1-11eb-81d5-ab94d43c0d3c.png)

**Reply by alemuntoni** [reply]
in the bottom-right area of the meshlab window, click on "show vertex quality histogram"

**Reply by DrBwts** [reply]
OK found it, it only changes the size of the max-min information. The values on the histogram scale are unaffected which is the part that I need to make bigger.

![image](https://user-images.githubusercontent.com/22524334/115067911-760c0580-9ee9-11eb-906c-ab4e6b3ca7f6.png)
